backend/clubs/management/commands/send_emails.py

```diff
@@ -121,11 +122,34 @@ def handle(self, *args, **kwargs):
         role_mapping = {k: v for k, v in Membership.ROLE_CHOICES}
 
         # handle custom Hub@Penn intro email
-        if action in {"hap_intro", "hap_intro_remind", "hap_second_round"}:
+        if action in {
+            "hap_intro",
+            "hap_intro_remind",
+            "hap_second_round",
+            "hap_partner_communication",
+        }:
             test_email = kwargs.get("test", None)
             email_file = kwargs["emails"]
             people = collections.defaultdict(dict)
 
+            if action == "hap_partner_communication":
+                emails = Club.objects.all().values_list("email", flat=True)
```

Comment:
This email is the public contact email, not the email of the resource owner. We want to get the email of the resource owner.

Reply:
Awesome thank you for this!